doc/040_backup.rst

```diff
@@ -240,6 +240,20 @@ include other filesystems like ``/sys`` and ``/proc``.
 .. note:: ``--one-file-system`` is currently unsupported on Windows, and will
     cause the backup to immediately fail with an error.
 
+Files larger than a given size can be excluded using the `--exclude-larger-than`
+option:
+
+.. code-block:: console
+
+    $ restic -r /srv/restic-repo backup ~/work --exclude-larger-than 1M
+
+This excludes files in ``~/work`` which are larger than 1 MB from the backup.
```

Comment:
Please add an empty line after this line :D

Reply:
OK:sweat_smile: